packages/coinstac-ui/app/render/components/dashboard-consortia.js

```diff
@@ -1,16 +1,39 @@
-import React, { PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { Button } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
 import ConsortiumCard from './consortium-card';
 import { connect } from 'react-redux';
-import { fetchConsortia } from '../state/ducks/consortia.js';
-import noop from 'lodash/noop';
+import {
+  deleteConsortium,
+  joinConsortium,
+  leaveConsortium,
+} from '../state/ducks/consortia';
 
-class DashboardConsortia extends React.Component {
+class DashboardConsortia extends Component {
+  renderConsortia() {
+    const { consortia, dispatch, user: { username } } = this.props;
 
-  componentWillMount() {
-    const { dispatch } = this.props;
-    dispatch(fetchConsortia(noop));
```

Comment:
Moved initial fetching to _bg-services_. We’re watching for new/changed consortia, which update the state accordingly.